week2/task9/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿var jupiterTime = new JupiterTime(3, 23);
+var titanTime = new TitanTime(1000, 23);
+var ganymedeTime = new GanymedeTime(200, 23);
+
+Console.WriteLine(jupiterTime);
+Console.WriteLine(titanTime);
+Console.WriteLine(ganymedeTime);
+
+public abstract class AlienTime
+{
+    private int _hours = 0, _minutes = 0, HoursInDays;
```

Comment:
a common convention for private fields is to have time starting with underscore and then lowercase, like _minutes. 
Therefore `HoursInDays` should be changed